client/lib/purchases/types.ts

```diff
@@ -270,9 +271,22 @@ export interface MembershipSubscriptionsSite {
 	subscriptions: MembershipSubscription[];
 }
 
-export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
-
 export interface Owner {
 	ID: string;
 	display_name: string;
 }
+export type GetChangePaymentMethodUrlFor = ( siteSlug: string, purchase: Purchase ) => string;
+export type GetManagePurchaseUrlFor = ( siteSlug: string, attachedToPurchaseId: string ) => string;
+
+export type RenderRenewsOrExpiresOn = (
+	moment: ReturnType< typeof useLocalizedMoment >,
+	purchase: Purchase,
+	siteSlug: string,
+	translate: ReturnType< typeof useTranslate >,
+	getManagePurchaseUrlFor: GetManagePurchaseUrlFor
+) => string;
+
+export type RenderRenewsOrExpiresOnLabel = (
+	purchase: Purchase,
+	translate: ReturnType< typeof useTranslate >
+) => string;
```

Comment:
The same is true of `renderRenewsOrExpiresOnLabel`; it accepts only one argument, an object.